src/home/views.py

```diff
@@ -101,6 +101,7 @@ def home_view(request):
             ),  # Add the rating_count field
             "Category": str(item.get("Category")),
             "MapLink": item.get("MapLink"),
+            "Description": item.get("Description", {}),  # Include Description
```

Comment:
nit: comment not needed. not a blocker

Reply:
Same as last comment, addressed in most recent commit